go.mod

```diff
@@ -5,10 +5,10 @@ go 1.19
 require (
 	github.com/gogo/protobuf v1.3.2
 	golang.org/x/tools v0.1.12
```

Comment:
Nit: would be good to align this with https://github.com/kubernetes/kubernetes/blob/release-1.26/go.mod#L87

Reply:
Done.